go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/xelaj/mtproto
+
+go 1.14
```

Comment:
Why not use the latest version 1.15?

Reply:
Wait, i have same question :thinking:

Reply:
This is just the Go version installed on my pc :)
As the [reference](https://golang.org/ref/mod#go-mod-file-go) says, the go version is used to determine which language features are available when compiling package.
I think this parameter is not so important. Many libraries use old versions of go in modules (because they don't use features from newer versions).
I can easily upgrade the version to the latest and refresh all dependencies if you want.